feature-libs/asm/customer-360/components/sections/asm-customer-map/asm-customer-map.component.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpParams } from '@angular/common/http';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { DomSanitizer, SafeResourceUrl } from '@angular/platform-browser';
+import {
+  Customer360SectionConfig,
+  Customer360StoreLocation,
+} from '@spartacus/asm/customer-360/root';
+import {
+  PointOfService,
+  TranslationService,
+  WeekdayOpeningDay,
+} from '@spartacus/core';
+import {
+  StoreFinderConfig,
+  StoreFinderSearchPage,
+  StoreFinderService,
+} from '@spartacus/storefinder/core';
+import { combineLatest, Observable, of, Subscription } from 'rxjs';
+import { concatMap, mapTo, take, tap } from 'rxjs/operators';
+
+import { Customer360SectionContext } from '../customer-360-section-context.model';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'cx-asm-customer-map',
+  templateUrl: './asm-customer-map.component.html',
+})
+export class AsmCustomerMapComponent implements OnDestroy, OnInit {
```

Comment:
[MINOR QUESTION]
Are we reusing any UI/services from the Storefinder library, e.g. to display list of stores, plug in the google maps, etc? Or we're "reinventing/copying" a lot of UI and logic to ASM library? Preferrably whe should reuse the business logic and _perhaps_ some reusable UI parts. What do you think about it?

PS. IMHO this question doesn't block the release of this feature. But it would be good to get an answer.

Reply:
We only utilize services from the Storefinder library. Displaying a map simply involves embedding an iframe where we provide a map URL. The design for displaying stores differs from ours, as our design features a list view, whereas Storefinder employs a tile view